dsp-value-objects/src/main/scala/dsp/valueobjects/List.scala

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright © 2021 - 2022 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package dsp.valueobjects
+
+import zio.prelude.Validation
+
+sealed trait List
+object List {
+
+  /**
+   * List ListName value object.
+   */
+  sealed abstract case class ListName private (value: String)
+  object ListName { self =>
+    def make(value: String): Validation[Throwable, ListName] =
+      if (value.isEmpty) {
+        Validation.fail(V2.BadRequestException(ListErrorMessages.ListNameMissing))
+      } else {
+        val validatedValue = Validation(
+          V2IriValidation.toSparqlEncodedString(value, throw V2.BadRequestException(ListErrorMessages.ListNameInvalid))
+        )
+
+        validatedValue.map(new ListName(_) {})
+      }
+
+    def make(value: Option[String]): Validation[Throwable, Option[ListName]] =
+      value match {
+        case Some(v) => self.make(v).map(Some(_))
+        case None    => Validation.succeed(None)
+      }
+  }
+
+  /**
+   * List Position value object.
+   */
+  sealed abstract case class Position private (value: Int)
+  object Position { self =>
+    def make(value: Int): Validation[Throwable, Position] =
+      if (value < -1) {
```

Comment:
According to the docs:
> In case the new node should be appended to the list of current children, either position: -1 must be given in the payload or the position parameter must be left out of the payload.

Position `0` must be the 1st possible position, like 0 in Arrays world is 1, 1 is 2 etc. I think there is an inconsistency in the error message.